Display a basic list of questions for a poll

matt989253 opened the pull request:
The DisplayPoll.Questions page is initialized with a hardcoded question list, showing that the displaying of questions is functional.
However, upon creating a question and fetching new ones, the hardcoded ones are replaced with the list of questions from the backend, which is empty.
Help welcome!

matt989253 committed:
Update Questions model

matt989253 committed:
Fix merge conflicts

matt989253 committed:
Merge remote-tracking branch 'upstream/master' into display-question

matt989253 committed:
Add a partial implementation to retrieve questions

matt989253 committed:
Merge remote-tracking branch 'upstream/master' into display-question

matt989253 committed:
Add a card to create a new question

matt989253 committed:
Add a semi-functional question list display, initialized with hardcoded questions

alexandrepiveteau commented:
The issue comes from the `Api/Questions` file. Apparently, the `create` method does not use the right endpoint and creates a new ... poll, rather than a question 😆 

This works because the `title` field is recognized by the server, and a new `Poll` is successfully inserted.

Please refactor the `Api/Questions` module to be similar to the `Api/Sessions` module, with a helper method that creates the endpoints (and minimizes the risk that we mess something up by mistake like here).

matt989253 committed:
Correct Question API, cleanup code

matt989253 commented:
Thanks for the help! I fixed the Question API and refactored it to be similar to the Sessions API.
I do get the expected answer from the server now, but i'm still getting an error that seems to happen during decoding or around that point. I'll keep snooping around

matt989253 committed:
Add a visible error display to the question

matt989253 committed:
Fix decoder error, remove debug code previously added

matt989253 committed:
Add a button to delete questions

matt989253 committed:
Cleanup imports
